### DISCUSSION TITLE
只有森林运行，庄园和其他没有运行，也没日志

### DISCUSSION THREAD

**Discussion by lzlq** [lazy-immortal/Sesame#18] [General]
仅森林运行，庄园和其他没有运行，也没日志。

**Comment by lazy-immortal** [comment]
设置-基础-只收能量时间

**Comment by lzlq** [comment]
thank you ！












劉旭
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年12月8日(周日) 上午9:41
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [lazy-immortal/Sesame] 只有森林运行，庄园和其他没有运行，也没日志 (Discussion #18)




 
设置-基础-只收能量时间
